### PR TITLE
fix defects reported by coverity scan

### PR DESCRIPTION
**Pull request by Jiaolong**

### CODE REVIEW

File: src/shogun/structure/GEMPLP.cpp [hunk 1]
```diff
@@ -266,6 +266,11 @@ float64_t CGEMPLP::inference(SGVector<int32_t> assignment)
 
 void CGEMPLP::update_messages(int32_t id_region)
 {
+	REQUIRE(m_factors != NULL, "Factors can not be empty! ");
```
Comment (karlnapf) [inline src/shogun/structure/GEMPLP.cpp]: newline missing, and no space at the end pls

File: src/shogun/structure/GEMPLP.cpp [hunk 2]
```diff
@@ -266,6 +266,11 @@ float64_t CGEMPLP::inference(SGVector<int32_t> assignment)
...
+	REQUIRE(m_factors->get_num_elements() > id_region, 
+			"Region id (%d) exceeds the factor elements' length (%d)!",
```
Comment (karlnapf) [inline src/shogun/structure/GEMPLP.cpp]: newline

**Comment by Jiaolong** [comment]
Thanks! I have corrected them.

**Comment by karlnapf** [comment]
Thanks!

**Comment by Jiaolong** [comment]
Thanks!

**Pull request merged**